src/htm/algorithms/SpatialPooler.cpp

```diff
@@ -703,7 +703,7 @@ Real SpatialPooler::avgConnectedSpanForColumnND_(UInt column) const {
 void SpatialPooler::adaptSynapses_(const SDR &input,
                                    const SDR &active) {
   for(const auto &column : active.getSparse()) {
-    connections_.adaptSegment(column, input, synPermActiveInc_, synPermInactiveDec_);
+    connections_.adaptSegment(column, input, synPermActiveInc_, synPermInactiveDec_, true, stimulusThreshold_);
```

Comment:
Q: make `stimulusThreshold_` a parameter of Connections? (SP has stimulusThreshold, TM has `minThreshold_` )

Reply:
I'd rather not move more logic into the connections class.  Besides, the SP and TM use the thresholds in different ways, which I think would greatly complicate this.

Reply:
I agree, seems better to leave the Conn's methods more configurable.